### ISSUE TITLE
Add sites

### ISSUE THREAD

**Issue by WebBreacher** [WebBreacher/WhatsMyName#250]
Examine https://github.com/soxoj/maigret/blob/main/maigret/resources/data.json and look for sites that could work in this project.

**Comment by snooppr** [comment]
Do not use this pirate project. Soxoj (Maigret) maliciously violated the copyright project [Snoop Project](https://github.com/snooppr/snoop). To do this, he acquired a subscription to Snoop Full Version, decompilaood the application and seized the database (the base was protected by encryption), and later included the database into his pirate project. Later after blocking, he began to comply with his code.
Do not allow you to mix a pirated project with an open source project. Read more about Piratv [here](https://github.com/jivoi/awesome-osint/issues/201).

**Comment by WebBreacher** [comment]
Thank you for letting me know. This project respects your copyrights and intellectual property.

Closing this issue.

**Comment by soxoj** [comment]
FYI, Maigret does not violate the copyright of Snoop Project, this complaint will be reviewed and rejected by GitHub, see my [DMCA counter-notice](https://github.com/github/dmca/blob/master/2020/11/2020-11-12-Snoop-counternotice.md).

**Comment by snooppr** [comment]
False. 
User purchased Snoop Full Version To access a secure database and illegally used the database in his pirate project (during locks From his mail (there is a payment and mail)), it was also blocked on Gitlab (DMCA was filed with respect to the Maigret pirate content, but Gitlab reviewed evidence has decided to block the decision Assembly account)
https://gitlab.com/soxoj
![изображение](https://user-images.githubusercontent.com/61022210/117672453-88a8f000-b1b2-11eb-999b-ed89924232da.png)

According to the Privacy Policy "GitHub DMCA" in the company accept complaints and counter-notifications at the same time, Github is not a judge and does not consider evidence A provides data for the possibility of a trial (such data I requested).
![git](https://user-images.githubusercontent.com/61022210/117673218-43d18900-b1b3-11eb-8585-df763cc92268.png)
Still in obtaining data on an attacker from GitHub

In addition, I squeezed the chats in Telegram with hashing (in case Soxoj starts cleaning the story, as he did / obfuscated with commits in Maigret, where Soxoj reports publicly that used BDFull Snoop Project) in which an attacker covered Snoop Project (Snoop database Project BDfull placed individual labels that, when you smoke copying, went to the data.json Maigret database).

Pictory Software and open source software are different universes. In modern society, pirate must be pursued and punished.
